src/group_theory/nilpotent.lean

```diff
@@ -377,3 +377,21 @@ begin
     apply mem_closure.mpr,
     exact λ K hK, hK ⟨f y, hd (mem_map_of_mem f hy), by simp⟩ }
 end
+
+lemma is_nilpotent_of_nilpotent_quotient_center
+  (hG : is_nilpotent (quotient_group.quotient (center G))) : is_nilpotent G :=
+begin
```

Comment:
Can you substantially golf this proof if you use `ker_le_center_to_nilpotent` from the other PR?